.github/integration-in-memory-tests.yml

```diff
@@ -76,6 +93,7 @@ runner-test-matrix:
     runs_on: ubuntu-latest
     triggers:
       - PR Integration CCIP Tests
+      - Nightly Integration CCIP Tests
```

Comment:
Whats the logic behind having some of these in the MQ and some not? Was this determined to be flakey?

Reply:
@kalverra can you please reply?